diff --git a/node_modules/node-osu/index.d.ts b/node_modules/node-osu/index.d.ts
index dbc8621..2ebd1df 100644
--- a/node_modules/node-osu/index.d.ts
+++ b/node_modules/node-osu/index.d.ts
@@ -131,6 +131,7 @@ export class Beatmap {
     hash: string;
     title: string;
     creator: string;
+    creatorId: string;
     version: string;
 
     source: string;
diff --git a/node_modules/node-osu/lib/base/Beatmap.js b/node_modules/node-osu/lib/base/Beatmap.js
index fe9028b..d4141f4 100644
--- a/node_modules/node-osu/lib/base/Beatmap.js
+++ b/node_modules/node-osu/lib/base/Beatmap.js
@@ -8,6 +8,7 @@ const Constants = require('../Constants.js');
  * @prop {String} hash
  * @prop {String} title
  * @prop {String} creator
+ * @prop {String|Number} creatorId
  * @prop {String} version
  * @prop {String} source
  * @prop {String} artist
@@ -57,6 +58,7 @@ class Beatmap {
 		this.hash = data.file_md5;
 		this.title = data.title;
 		this.creator = data.creator;
+		this.creatorId = data.creator_id;
 		this.version = data.version;
 
 		this.source = data.source;
